File: .vscode/Java_Programs/Q10.java
//Problem Statement
//Write a Java program to check whether a given number is prime


import java.util.Scanner;

class PrimeCheck {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num;
        boolean isPrime = true;
        System.out.println("Enter a number: ");
        num = sc.nextInt();

        for(int i = 2; i <= num/2; i++) { 
            if(num % i = 0) { 
                isPrime = false;
                break;
            }
        }

        if(isPrime)
            System.out.println(num + " is a Prime Number");
        else
            System.out.println(num + " is not a Prime Number");

        sc.close();
    }
}

//Expected Output
        //Enter a number: 7
        //7 is a Prime Number

File: .vscode/C-Programs/Q10.c
//Problem Statement
//Write a C program to generate the Fibonacci series up to n terms.

#include <stdio.h>

int main() {
    int n, a = 0, b = 1, c, i;
    printf("Enter the number of terms: ");
    scanf("%d", n); 

    printf("%d %d ", a, b);
    for(i = 2; i < n; i++) { 
        c = a + b;
        printf("%d ", c);
        a = b;
        b = c;
    }
    return 0;
}

//Expected Output
        //Enter the number of terms: 5
        //0 1 1 2 3

